tests/test_backends.py

```diff
@@ -71,21 +76,15 @@ def test_natives_no_cz_cnot():
         "GPI2",
         "GPI",
         "M",
-        "CZ",
-        "CNOT",
     }
 
-    for gate in ["CZ", "CNOT"]:
-        for p in platform.pairs:
-            setattr(platform.pairs[p].native_gates, gate, None)
-        assert gate not in set(backend.natives)
-
```

Comment:
Maybe something got wrong in the conflicts' resolution. I'd reset it to the state of #1076. Do you agree @csookim?

Reply:
Because the issue was only in `def test_natives_no_cz_cnot()`, I fixed it manually.